lambdas/cis2-api/src/utils/logger.ts

```diff
@@ -0,0 +1,15 @@
+import winston from 'winston';
+
+const { combine, timestamp, json } = winston.format;
+
+const logger = winston.createLogger({
+  level: 'info',
+  format: combine(timestamp(), json()),
```

Comment:
```suggestion
  format: combine(errors({ stack: true }), timestamp(), json()),
```
We're not logging any errors yet, but might be forgotten if someone adds an error log in the future. It wouldn't occur to me.

Reply:
updated